ProcessSlave.php

```diff
@@ -332,46 +337,48 @@ public function commandBootstrap(array $data, Connection $conn)
     /**
      * Handles incoming requests and transforms a $request into a $response by reference.
      *
-     * @param \React\Http\Request $request
-     * @param HttpResponse        $response
+     * @param RequestInterface $request
      *
+     * @return ResponseInterface
      * @throws \Exception
      */
-    public function onRequest(\React\Http\Request $request, HttpResponse $response)
+    public function onRequest(RequestInterface $request)
     {
         $this->prepareEnvironment($request);
 
+        $response = $this->handleRequest($request);
```

Comment:
Would it be good to add a catch(\Throwable) here, log to master and potentially close the slave?